src/ocean/text/util/StringC.d

```diff
@@ -89,7 +89,9 @@ class StringC
     {
         if (str.length && !!str[$ - 1])
         {
+            enableStomping(str);
             str ~= StringC.Term;
```

Comment:
> The reasoning you remember was mostly related to ocean.core.array and similar widely used array utilities where it is justified by amount of porting effort it saves. In general, calling enableStomping upon changing length is expected from downstream apps same as with libraries.


Yeah, that's right, and it makes sense.

> Not that appending (like str -= StringC.Term) is what I refer to as "changing length" when talking about stomping stuff. Only explicit assignment to .length property require enableStomping (and only if they may shrink length).

Ha, yes, I missed the point of shrinking length. But, is then this PR even necessary? I don't think so?